### ISSUE TITLE
Publishing Extensions guide - Token permissions and VSCE command changed

### ISSUE THREAD

**Issue by robole** [microsoft/vscode-docs#4218]
There are some mistakes in this guide: https://code.visualstudio.com/api/working-with-extensions/publishing-extension 

1. The personal access token (PAT) you create needs full permissions and access to all organization currently. This is a longstanding issue (I encountered this 8 months ago also). It is discussed in [this issue](https://github.com/microsoft/vscode-vsce/issues/522) in the VSCE repo.
2. `vsce create-publisher (publisher name)` is no longer available in VSCE. You must go to the Marketplace publisher management page to create a publisher.

![err](https://user-images.githubusercontent.com/3703647/103671399-d10e9200-4fb5-11eb-9d5c-54b07905aac2.png)

I can make a PR if you wish?

**Comment by joaomoreno** [comment]
> I can make a PR if you wish?

Absolutely, we'd love to have your contribution!

**Comment by joaomoreno** [comment]
I don't think full permissons are needed. **Marketplace > Manage** should be the only scope necessary, so I adjusted the PR accordingly. I find it strange that **all organizations** was missing from the docs, since I clearly remember adding it. 🤔

**Comment by robole** [comment]
@joaomoreno The problem is the process contains arbitrary steps. I believe _all organizations_ was mentioned in the description before, but it was not in the screenshot. My edit was to draw more attention to it. 

When I went through this process the first time, I did not use azure before. The user guide is written from the POV that you have used azure before. I had to create an organization for the first time. 

When it comes to selecting an organization for the PAT, I must have picked my own organization. If you only have 1 organization, why pick _all organizations_ after all? 

I repeated the process twice I believe and the only outcome that I recall working was by having _all organizations_ and _all scopes_ applied also. So while the guide can urge people to follow the instructions blow-by-blow, it is a strange path to follow to set things up.
